spec/unit/harvestdor-indexer_spec.rb

```diff
@@ -119,7 +119,7 @@
       end
     end
     context "load_whitelist" do
-      it "should not be called if there was no whitelist config setting" do
+      it "not called if there was no whitelist config setting" do
```

Comment:
Did you get these because of "rubocop -a"'s default behavior (with rubocop-rspec turned on)? Just curious, as I've found this block to be helpful in getting rubocop-rspec to do the right thing: https://github.com/projecthydra/sufia/blob/master/.rubocop.yml#L132-L139